libuavcan/include/uavcan/transport/can_io.hpp

```diff
@@ -1,15 +1,18 @@
 /*
  * CAN bus IO logic.
  * Copyright (C) 2014 Pavel Kirienko <pavel.kirienko@gmail.com>
+ * Copyright (C) 2019 Theodoros Ntakouris <zarkopafilis@gmail.com>
  */
 
 #ifndef UAVCAN_TRANSPORT_CAN_IO_HPP_INCLUDED
 #define UAVCAN_TRANSPORT_CAN_IO_HPP_INCLUDED
 
 #include <cassert>
+#include <utility>
```

Comment:
One small include for [a] man, one large change in the C++ compiler requirements: https://uavcan.org/Implementations/Libuavcan/#c-compiler-requirements. Can we cope without this include, or at least make it conditional on C++11?